encodings/json/README.md

```diff
@@ -0,0 +1,30 @@
+# OCSF JSON Encoding
+
+Since OCSF was developed from a JSON perspective, there are not
+a lot of things to call out here for best practices.
+
+For example, the [OCSF server](https://github.com/ocsf/ocsf-server)
+already contains tooling to export the schema in [JSON
+Schema](https://json-schema.org/) format, which can be used
+to automate validation of JSON-encoded OCSF events.
+
+## Semantic Type Considerations
```

Comment:
Commenting on the use of [Object](https://schema.ocsf.io/1.1.0/objects/object?extensions=), particularly for unmapped data would probably be helpful.

Reply:
That's been a topic of some discussion, for sure.  In particular, it's caused me some grief because it's the one place where it's basically _required_ that undefined attributes be present.

In the context of JSON encoding, it would definitely be worth calling out, I'll add that :+1: